pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
Calling it "agile" is just using a buzzword to justify creating technical debt by not considering the general storage picture. There's no time critical reason why this feature must be pushed in right now rather than providing a general solution, which is barely any more work than what's done here. so I'm still against merging this as it is proposed here.